chromium_src/chrome/browser/ui/views/frame/tab_strip_region_view.h

```diff
@@ -0,0 +1,9 @@
+/* Copyright (c) 2022 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#define BRAVE_UI_VIEWS_FRAME_TAB_STRIP_REGION_FRIEND \
```

Comment:
With below, we don't need to add corresponding patch files.
```
#define FrameColorsChanged                   \
  UnUsed() {}                                \
  friend class BraveTabStripRegionContainer; \
  void FrameColorsChanged
#include "src/chrome/browser/ui/views/frame/tab_strip_region_view.h"
#undef FrameColorsChanged
```

Reply:
Cool! done.